packages/labs/ssr/package.json

```diff
@@ -228,7 +228,7 @@
     "@lit-labs/ssr-dom-shim": "^1.2.0",
     "@lit/reactive-element": "^2.0.4",
     "@parse5/tools": "^0.3.0",
-    "@types/node": "^16.0.0",
+    "@types/node": ">=18.0.0",
```

Comment:
perhaps this should be a dev dependency instead? i think moving it devDependencies would make it not install for downstream consumers.

Reply:
Yes, but I wonder if that would break consumers? But I guess if consumers use TypeScript for this, they would have @types/node already installed?

Reply:
Hm, `render-result-readable.ts` imports/exports `Readable` as part of the public API. I will test if that breaks, when using that in a consuming project and running type checking.

Reply:
Ok, did a quick check. When we move it to a devDependency, it works when `skipLibCheck` in tsconfig.json is set to true. Otherwise it breaks the build.
Is that ok?
Alternatively we could also define it as an optional peer dependency.